runelite-client/src/main/java/net/runelite/client/plugins/twitch/TwitchPlugin.java

```diff
@@ -226,4 +295,111 @@ public boolean onPrivateMessageInput(PrivateMessageInput privateMessageInput)
 	{
 		return false;
 	}
+
+
+	@Subscribe
+	private void onMenuOptionClicked(MenuOptionClicked event)
+	{
+		if (event.getWidgetId() != CLAN_BUTTON_ID)
+		{
+			return;
+		}
+
+		if (!event.getMenuOption().equals("Switch tab") && !event.getMenuOption().equals("Switch:") &&
+			!event.getMenuOption().equals(TWITCH_FILTER_MENUOPTION))
```

Comment:
Probably personal preference but doing a switch on event.getMenuOption() would be cleaner. Would also prevent the duplicate checks below.

You may also want to move these strings to static variables since they are used in multiple places